04_UpdateUser.php

```diff
@@ -2,106 +2,104 @@
+
+				<!-- Float links to the right -->
+				<div class="w3-right" >
+					<a href="indexV1.php" class="w3-bar-item w3-button w3-xxlarge fa fa-home"></a>
+					<a href="playlistV1.php" class="w3-bar-item w3-button w3-xxlarge material-icons">playlist_add</a>	
+					<a href="contact.php" class="w3-bar-item w3-button w3-xxlarge material-icons" >contact_support</a>
+					<a href="settings.php" class="w3-bar-item w3-button w3-xxlarge material-icons">settings</a>	
 				</div>
 			</div>
-		
-		
+		</div>
+
+
 		<br>
 		<br>
 		<br>
 
 
-			<!-- side navbar for admin + php codes -->
-			<div class="w3-sidebar w3-bar-block w3-border-right" style="width:8%">
-				<a href="03_add_user(settings).php" class="w3-bar-item w3-button w3-xxlarge material-icons">person_add</a> 
-				<a href="02_show_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">visibility</a> 
-			 	<a href="04_update_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">manage_accounts</a>
-			  	<a href="05_delete_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">person_remove</a>
+		<!-- side navbar for admin + php codes -->
+		<div class="w3-sidebar w3-bar-block w3-border-right" style="width:8%">
+			<a href="03_add_user(settings).php" class="w3-bar-item w3-button w3-xxlarge material-icons">person_add</a> 
+			<a href="02_show_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">visibility</a> 
+			<a href="04_update_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">manage_accounts</a>
+			<a href="05_delete_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">person_remove</a>
 
+		</div>
+
+
+
+		<div style="margin-left:8%">
+			<div class="w3-container w3-dark-grey">
+				<h2>Update User</h2>
 			</div>
-		
 
+			<div class="w3-container">
+				<div class="content"><!-- Holds the update users content -->
 
-			<div style="margin-left:8%">
-				<div class="w3-container w3-dark-grey">
-					<h1>Update User</h1>
-				</div>
+					<form method = "post" id= "05_update_user.php">
+						<h6><label for = 'login'>Existing User Name:</label>
+							<input type = "text" name = "ExistingUserName" placeholder="Enter existing user name"/></h6>
+						<h6><label for = 'login'>New User Name:</label>
+							<input type = "text" name = "NewUserName" placeholder="Enter new user name"/></h6>
+						<h6><input type = "submit" value = "Update" /></h6><br>
+					</form>
+
+					<?php
+					require "communityproject_mysqli.php";
+					print "<p class = 'yellow'>Connected to server</p>";
+
+					$ExistingUserID = isset($_POST['ExistingUserName']) ?$_POST['ExistingUserName']: '';
+					$NewUserID =  isset($_POST['NewUserName']) ?$_POST['NewUserName']: '';
+
+					//create a variable to store sql code for the 'Add Users' query
+					$updatequery = "UPDATE users SET User_ID = '$NewUserID' WHERE User_ID = '$ExistingUserID'";
+					if (mysqli_query($conn, $updatequery))
+					{
+						echo "<p>Record updated</p>";
```

Comment:
changed h3 to p as h3 was too big for the page and p kept the theme consistent and professional